### PR TITLE
optimizer: modify docs for analyze behavior

### PR DESCRIPTION
**Pull request by winoros**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

We've updated the analyze behavior when setting `tidb_analyze_version=2` and change its default value to 2.

This pr is to make the doc compatible with it.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: statistics.md [hunk 1]
```diff
@@ -61,6 +61,7 @@ Top-N 即是这个列或者这个索引中，出现次数前 n 的值。TiDB 会
 > 如需更快的分析速度，可将 `tidb_enable_fast_analyze` 设置为 `1` 来打开快速分析功能。该参数的默认值为 `0`。
 >
 > 快速分析功能开启后，TiDB 会随机采样约 10000 行的数据来构建统计信息。因此在数据分布不均匀或者数据量比较少的情况下，统计信息的准确度会比较差。可能导致执行计划不优，比如选错索引。如果可以接受普通 `ANALYZE` 语句的执行时间，则推荐关闭快速分析功能。
+> `tidb_enable_fast_analyze` 为实验性功能，目前并不与 `tidb_analyze_version=2` 的统计信息完全匹配，因此设置 `tidb_enable_fast_analyze` 请将 `tidb_analyze_version` 设置为 1。
```
Comment (time-and-fate) [inline statistics.md:64]: > 设置 `tidb_enable_fast_analyze` 请将 `tidb_analyze_version` 设置为 1

=>

> 开启 `tidb_enable_fast_analyze` 时请将 `tidb_analyze_version` 设置为 1

感觉这样通顺一点吧

**Review by time-and-fate — APPROVED** [review]
/lgtm

File: statistics.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ aliases: ['/docs-cn/dev/statistics/','/docs-cn/dev/reference/performance/statist
 
 # 统计信息简介
 
-TiDB 使用统计信息来决定[索引的选择](/choose-index.md)。变量 `tidb_analyze_version` 用于控制所收集到的统计信息。目前 TiDB 中支持两种统计信息：`tidb_analyze_version = 1` （默认）以及 `tidb_analyze_version = 2`。两种版本中，TiDB 维护的统计信息如下：
+TiDB 使用统计信息来决定[索引的选择](/choose-index.md)。变量 `tidb_analyze_version` 用于控制所收集到的统计信息。目前 TiDB 中支持两种统计信息：`tidb_analyze_version = 1` 以及 `tidb_analyze_version = 2`。在 5.1 以前的版本中，该变量的默认值为 1，在 5.1 中，该变量的默认值为 2，作为实验性特性启用。两种版本中，TiDB 维护的统计信息如下：
```
Comment (TomShawn) [inline statistics.md:8]: 是“在 v5.1 中”还是“自 v5.1 起”呢？
```suggestion
TiDB 使用统计信息来决定[索引的选择](/choose-index.md)。变量 `tidb_analyze_version` 用于控制所收集到的统计信息。目前 TiDB 中支持两种统计信息：`tidb_analyze_version = 1` 以及 `tidb_analyze_version = 2`。在 v5.1 以前的版本中，该变量的默认值为 `1`。在 v5.1 中，该变量的默认值为 `2`，作为实验特性启用。两种版本中，TiDB 维护的统计信息如下：
```

File: statistics.md [hunk 3]
```diff
@@ -61,6 +61,7 @@ Top-N 即是这个列或者这个索引中，出现次数前 n 的值。TiDB 会
 > 如需更快的分析速度，可将 `tidb_enable_fast_analyze` 设置为 `1` 来打开快速分析功能。该参数的默认值为 `0`。
 >
 > 快速分析功能开启后，TiDB 会随机采样约 10000 行的数据来构建统计信息。因此在数据分布不均匀或者数据量比较少的情况下，统计信息的准确度会比较差。可能导致执行计划不优，比如选错索引。如果可以接受普通 `ANALYZE` 语句的执行时间，则推荐关闭快速分析功能。
+> `tidb_enable_fast_analyze` 为实验性功能，目前并不与 `tidb_analyze_version=2` 的统计信息完全匹配，因此开启 `tidb_enable_fast_analyze` 时请将 `tidb_analyze_version` 设置为 1。
```
Comment (TomShawn) [inline statistics.md:64]:
```suggestion
>
> `tidb_enable_fast_analyze` 为实验性功能，目前与 `tidb_analyze_version=2` 的统计信息**不完全匹配**。因此开启 `tidb_enable_fast_analyze` 时需要将 `tidb_analyze_version` 的值设置为 `1`。
```

File: statistics.md (hunk 2 above)
Comment (time-and-fate) [reply statistics.md:8]: 自 v5.1 起

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**